### PR TITLE
Add signaling server to issue template

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -18,6 +18,17 @@ Tell us what should happen
 ### Actual behaviour
 Tell us what happens instead
 
+## Talk app
+
+**Talk app version:** (see apps admin page: `/index.php/settings/apps`)
+
+**Custom Signaling server configured:** yes/no (see additional admin settings: `/index.php/settings/admin/additional`)
```
Comment (danxuliu) [inline .github/ISSUE_TEMPLATE/bug_report.md:25]: Shouldn't this be `/index.php/settings/admin/talk#signaling_server` or, if that is too specific, `/index.php/settings/admin/talk`?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -18,6 +18,17 @@ Tell us what should happen
...
+
+**Custom TURN server configured:** yes/no (see additional admin settings: `/index.php/settings/admin/additional`)
```
Comment (danxuliu) [inline .github/ISSUE_TEMPLATE/bug_report.md:27]: Shouldn't this be `/index.php/settings/admin/talk#turn_server` or, if that is too specific, `/index.php/settings/admin/talk`?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -18,6 +18,17 @@ Tell us what should happen
...
+
+**Custom STUN server configured:** yes/no (see additional admin settings: `/index.php/settings/admin/additional`)
```
Comment (danxuliu) [inline .github/ISSUE_TEMPLATE/bug_report.md:29]: Shouldn't this be `/index.php/settings/admin/talk#stun_server` or, if that is too specific, `/index.php/settings/admin/talk`?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (nickvergessen) [reply .github/ISSUE_TEMPLATE/bug_report.md:25]: Since the page is "short enough" and we need answers to all 3 questions anyway I guess it's fine?

**Pull request merged**